harness/determined/layers/_workload_sequencer.py

```diff
@@ -87,17 +87,17 @@ def __init__(
 
     def __init__(
         self,
-        generic_context: _generic.Context,
+        core_context: _core.Context,
         env: det.EnvContext,
     ) -> None:
-        self.generic = generic_context
+        self.core = core_context
```

Comment:
maybe it'd be better for readability to do:

```suggestion
        self.core_context = core_context
```